src/workload.py

```diff
@@ -174,21 +170,21 @@ def cleanup_after_pod_restart(self) -> None:
     def _bootstrap_router(self, *, tls: bool) -> None:
         """Bootstrap MySQL Router and enable service."""
         logger.debug(
-            f"Bootstrapping router {tls=}, {self._database_requires_relation.host=}, {self._database_requires_relation.port=}"
+            f"Bootstrapping router {tls=}, {self._connection_info.host=}, {self._connection_info.port=}"
         )
         try:
             # Bootstrap MySQL Router
             process = self._container.exec(
                 [
                     "mysqlrouter",
                     "--bootstrap",
-                    self._database_requires_relation.username
+                    self._connection_info.username
                     + ":"
-                    + self._database_requires_relation.password
+                    + self._connection_info.password
                     + "@"
-                    + self._database_requires_relation.host
+                    + self._connection_info.host
                     + ":"
```

Comment:
Is any good reason why we parse host and port from the relation and add an additional level of abstraction in src/relations/database_requires.py to concat host and port back in one real place where we use it?